inventory/by_cloud/aws

```diff
@@ -1 +1,16 @@
-# to be used for AWS resources
\ No newline at end of file
+[production:children]
```

Comment:
Test to make sure this syntax is additive, and the `production` group will contain these groups AND the ones from `inventory/by_environment`.